### DISCUSSION TITLE
How can I change the automatic backup start time/hour?

### DISCUSSION THREAD

**Discussion by luzfcb** [nextcloud/all-in-one#4248] [Questions] [answered]
Hello, How can I change the automatic backup start time/hour?

My physical server is set to UTC−03:00. This means that the Nextcloud Master container will always be 3 hours ahead, since inside the container it is always UTC time.

In other words, if outside the docker container it is "Mon Feb 19 10:00:00 -03 2024", inside the Nextcloud Master Container it will be "Mon Feb 19 13:00:00 UTC 2024"

The problem is that the backup is starting at an inconvenient time, close to 9:00 in UTC−03:00.

**Comment by szaimen** [comment, marked as answer]
Hi, simply click on `disable daily backups`. Then you have the choice to set a new time.
